src/sections/collection/datasets-list/DatasetsList.module.scss

```diff
@@ -1,13 +1,45 @@
-@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module";
+@import 'node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module';
 
 .container {
   min-height: calc(100vh + 100px);
-  padding:15px;
+  padding: 15px;
```

Comment:
I have this separated on my end, perhaps there is some miss configuration

Reply:
When I execute `npm run lint:stylelint`, it does not correct the spacing, which suggests there might be an issue with the Stylelint configuration. If there's a quick solution, we can implement it in this PR. Otherwise, we should create a new issue

Reply:
- #345